### ISSUE TITLE
optional pre/postfix creating the resource group

### ISSUE THREAD

**Issue by tdemeester** [test-kitchen/kitchen-azurerm#50]
it would be nice to be able to pre/postfix the resource group name so it can comply with naming standards within an organisation. could be extended to support further naming conventions on other resources.

thought for solution:
could be via a config file in the .azure folder

**Comment by tdemeester** [comment]
in code i found you can change the resourcegroupname using the driver_config ```azure_resource_group_name```

I will update the readme with an example of this.

**Comment by stuartpreston** [comment]
Good idea, I'll leave this on the backlog for now

**Comment by stuartpreston** [comment]
This will be available in the next release 0.12.  You'll be able to do something like the following (as well as overriding `azure_resource_group_name` if you wish (though I recommend not doing so because it is calculated from the suites you have in your config):

```
driver_config:
  azure_resource_group_prefix: sp-
  azure_resource_group_suffix: -DEVELOPMENT
```

![image](https://user-images.githubusercontent.com/6384654/29380234-f69b16e8-82bc-11e7-932e-c568151e92d5.png)

**Comment by stuartpreston** [comment]
This has now been released in 0.12 - please update your kitchen-azurerm to make use of it
